minicriu-client.c

```diff
@@ -100,6 +100,52 @@ static int writefile(const char *file, const char *buf, size_t len) {
 	return bytes;
 }
 
+static void mc_sighnd(int sig) {
```

Comment:
Let's avoid moving the function and instead provide a declaration on top of the file